steering_controllers_library/include/steering_controllers_library/steering_odometry.hpp

```diff
@@ -116,6 +116,8 @@ class [[deprecated("Use steering_kinematics::SteeringKinematics instead")]] Stee
 
   void reset_odometry();
 
+  bool try_update_open_loop(double linear, double angular, double delTime);
```

Comment:
I'm not sure about the naming as we will always have a successful update call, but it may or maynot fail, so not sure about the naming here

@christophfroehlich what do you think about adding this new method? or we simply change the void return type to bool and have it all¿+?